packages/itwinui-react/package.json

```diff
@@ -90,7 +90,6 @@
     "@typescript-eslint/parser": "^5.17.0",
     "babel-loader": "^8.2.2",
     "concurrently": "^5.3.0",
-    "configs": "*",
```

Comment:
What is the meaning of `*`? Does it mean any version?

Reply:
yes. 

i removed it because changesets was complaining about it being an "ignored" package in the config